src/Okta.Sdk/Generated/SamlApplicationSettingsSignOn.Generated.cs

```diff
@@ -16,6 +16,20 @@ namespace Okta.Sdk
     /// <inheritdoc/>
     public sealed partial class SamlApplicationSettingsSignOn : Resource, ISamlApplicationSettingsSignOn
     {
+        /// <inheritdoc/>
+        public IList<IAcsEndpoint> AcsEndpoints 
```

Comment:
Did these changes come with the latest version of the openapi spec?

Reply:
I shouldn't have included this in the PR, this is not related to the issue. I'll remove it, thanks.

Reply:
It's OK, this could happen when you run the generation script using the latest version of the spec. If changes are small, I'm OK including them on the same PR, otherwise, we should have a separate PR with the latest spec changes.

Reply:
Yes, there were several other changes to generated files not related to the issue, I excluded them from the initial PR but missed this one. 
I think we should not put code changes done in scope of different stories in one PR. So I removed this changes.